### PR TITLE
firmware: Add DAPLink beta hex files and info.

### PR DESCRIPTION
**Pull request by microbit-carlos**
@jaustin Adding the kl26 (still needs to be tested) and kl27 DAPLink GCC betas. I'm assuming this is the best place to put them.

I kept the description relatively short, let me know if we should change or add something.

We can then send a newsletter pointing to this.

### CODE REVIEW

**Comment by microbit-carlos** [comment]
The KL26 beta does have some issues that block us from releasing it as a beta, so I've removed it from the PR.

@microbit-robert when do you think you might have some time to review this PR? We were thinking about sending the newsletter Tuesday or Thursday.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+Icon?
+ehthumbs.db
+Thumbs.db
+Desktop.ini
+$RECYCLE.BIN/
```
Comment (microbit-carlos) [inline .gitignore:11]: Normally most people should have these in their global .gitignore, but it doesn't hurt to have them here as well.

**Review by microbit-robert — APPROVED** [review]
All looks good, thanks.

**Comment by microbit-carlos** [comment]
Thanks @microbit-robert!

**Pull request merged**